build.gradle

```diff
@@ -64,6 +65,10 @@ pluginBundle {
     }
 }
 
+java {
+    withSourcesJar()
```

Comment:
I thought it would be nice to publish sources too (`-sources.jar` artifact would be added to maven publicatioN)
![image](https://user-images.githubusercontent.com/36954793/72422517-ea09ac00-3782-11ea-97dd-c1324e0c08bf.png)